src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -82,6 +82,10 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
         <service id="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" alias="serializer.normalizer.object" />
+        
+        <service id="serializer.normalizer.flatten_exception" class="Symfony\Component\Serializer\Normalizer\FlattenExceptionNormalizer">
+            <tag name="serializer.normalizer" priority="-800" />
```

Comment:
I had to put -800 so it is called before ObjectNormalizer. If I put -915 as suggested, the right normalized it's not called, so it will be creating a wrong json.